p2p/peer_set.go

```diff
@@ -50,10 +50,12 @@ func (ps *PeerSet) Add(peer *Peer) error {
 	return nil
 }
 
+// Has returns true iff the peerset contains
+// the peer referred to by this peerKey.
 func (ps *PeerSet) Has(peerKey string) bool {
```

Comment:
Would you mind adding proper comments to all the functions in this file, please ?

Reply:
For sure, great suggestion, thanks!

Reply:
I added comments to all the functions in this file.